pkg/volume/volume_linux.go

```diff
@@ -74,7 +77,9 @@ func SetVolumeOwnership(mounter Mounter, fsGroup *int64, fsGroupChangePolicy *v1
 		return changeFilePermission(path, fsGroup, mounter.GetAttributes().ReadOnly, info)
 	})
 	if completeFunc != nil {
-		completeFunc(&err)
+		completeFunc(types.CompleteFuncParam{
+			Err: &err,
+		})
```

Comment:
Maybe less important but do we also want to pass in migrated info here for consistency? How hard is it to get the migrated field here?

Reply:
Hmm, I might be wrong but this seems not to be a migrated operation? That being said, there is no difference in in-tree/CSI for this operation. So by default it will be migrated:false